src/pgzero/storage.py

```diff
@@ -189,3 +218,39 @@ def _get_json_error_keys(cls, obj, json_path='storage'):
 
 
 storage = Storage()
+
+
+# This function is used to create the screenshot instance with the file name
+# given by runner.py but save it in the scope of storage.
+def _initialize_screenshots(file_path):
+    # If the instance already exists, do nothing.
+    if "screenshots" in globals():
+        return
+    # Otherwise, create the instance of the Screenshots class used to
+    # take and save screenshots.
+    if not os.path.isabs(file_path):
+        file_path = os.path.abspath(file_path)
+    project_name, _ = os.path.splitext(os.path.basename(file_path))
+
+    global screenshots
+    screenshots = Screenshots(project_name)
+
+
+class Screenshots:
+    """Class to manage taking screenshots."""
+    def __init__(self, project_name):
+        self._project_name = project_name
+        self._path = _get_platform_screenshot_path()
+
+    def take(self, surface):
+        # Ensure that the directory for screenshots exists.
+        _ensure_path(self._path)
+
+        # Creates the filename, made up of the script name and a timestamp.
+        timestamp = datetime.now().strftime("-%y%m%d-%H%M%S")
```

Comment:
```suggestion
        timestamp = datetime.now().strftime("-%Y-%m-%d_%H:%M:%S")
```

This is ISO format and more readable for humans.

Reply:
Fixed.